simulator-core/src/test/java/com/evbox/everon/ocpp/mock/factory/SimulatorConfigCreator.java

```diff
@@ -21,6 +22,23 @@ public static SimulatorConfiguration createSimulatorConfiguration(SimulatorConfi
 
         stationConfiguration.setId(stationId);
         stationConfiguration.setEvse(evse);
+        stationConfiguration.setBasicAuthPassword(BASIC_AUTH_PASSWORD);
+        return stationConfiguration;
+    }
+
+    public static SimulatorConfiguration.StationConfiguration createStationConfiguration(String stationId,
+                                                                                         int evseCount,
```

Comment:
this indentation looks really strange, at least it's the first time I'm seeing it in our code base, could you run it through the formatter?

Reply:
it is formatted accordingly.